src/Routes.php

```diff
@@ -135,6 +135,8 @@ public function create():array
                 ->withTrashed($route['with-trashed']);
         }
 
+        $this->router->getRoutes()->refreshNameLookups();
```

Comment:
I didn't know about this.. thanks. Would it work the same if we add it the test after the routes are defined?

Might let us avoid including test specific code in the implementation.

Reply:
Sorry I didn't understand the question. What I found is, once the routes are changed or all routes are defined, then we need to do a refresh name lookup.

so we should do it after all the routes are defined or when we add a dynamic routes (this is the way it is right now)

Reply:
@mitulgolakiya thats cool, can add this line in the test after you define the routes though

We don't need it in the class because this only effects testing